litgpt/finetune/adapter.py

```diff
@@ -69,6 +72,7 @@ def setup(
         data: Data-related arguments. If not provided, the default is ``litgpt.data.Alpaca``.
         train: Training-related arguments. See ``litgpt.args.TrainArgs`` for details.
         eval: Evaluation-related arguments. See ``litgpt.args.EvalArgs`` for details.
+        optimizer: An optimizer such as torch.optim.AdamW.
```

Comment:
To be applied to the other files too:

```suggestion
        optimizer: An optimizer name (such as "AdamW") or config.
```